examples/Main.hs

```diff
@@ -0,0 +1,8 @@
+module Main where
+
+import qualified DiffieHellman
+import qualified Example
+import Protolude
+
+main :: IO ()
+main = pure ()
```

Comment:
Maybe pass the `main` functionality of `examples/DiffieHellman` here rather than doing nothing?

Reply:
It just needs the status code to be 0 for the test to pass. All this is doing really is checking that the examples modules compiles against the latest version of the library.